### PR TITLE
Bump timescaledb to 2.8.1

### PR DESCRIPTION
**Pull request by hughcapet**
Bump timescaledb to 2.8.1

### CODE REVIEW

File: postgres-appliance/Dockerfile [hunk 1]
```diff
@@ -286,27 +287,6 @@ RUN export DEBIAN_FRONTEND=noninteractive \
...
-        fi \
-        && pg_buildext updatecontrol \
-        && DEB_BUILD_OPTIONS=nocheck debuild -b -uc -us \
-        && cd .. \
-        && for version in $DEB_PG_SUPPORTED_VERSIONS; do \
-            for deb in postgresql-${version}-${pkg}_*.deb; do \
-                if [ -f $deb ]; then dpkg -i $deb; fi; \
-            done; \
-        done; \
-    done \
```
Comment (FxKu) [inline postgres-appliance/Dockerfile:309]: PR description could have a note why this part could be replaced with the one line above.

Comment (hughcapet) [reply postgres-appliance/Dockerfile:309]: Ok, I will add it to the PR that introduced this change

Comment (FxKu) [reply postgres-appliance/Dockerfile:309]: Ok, I see that #786 gives the answer (somewhat)

**Comment by FxKu** [comment]
:+1:

**Pull request merged**